packages/forms-web-app/src/controllers/projects/examination-timetable.js

```diff
@@ -31,13 +31,12 @@ const eventStates = [
 ];
 
 const eventSubmitButtonActive = (timetable) => {
-	const tomorrow = new Date();
-	tomorrow.setDate(tomorrow.getDate() + 1);
+	const currentDate = new Date();
 
 	return (
 		timetable.typeOfEvent === 'Deadline' &&
-		tomorrow < new Date(timetable.dateOfEvent) &&
-		(new Date(timetable.dateTimeDeadlineStart) < getDate() ||
+		currentDate < new Date(timetable.dateOfEvent).setHours(24, 0, 0, 0) &&
```

Comment:
can this be a func  - 
`const dateIn24Hours = (date) => new Date(date).setHours(23, 0, 0, 0);`

occurs twice so would be nice to lift.

Reply:
### or something better named. 

and used in the two places instead.

Reply:
Done - added setTimeToEndOfDay.